src/BlockTypes/MiniCart.php

```diff
@@ -192,6 +192,7 @@ protected function get_markup() {
 		$cart                = $cart_controller->get_cart_instance();
 		$cart_contents_count = $cart->get_cart_contents_count();
 		$cart_contents       = $cart->get_cart();
+		$cart_contents_total = $cart->get_cart_contents_total();
```

Comment:
This has the same issue as the React component, if you set the settings as described in my previous message, this price doesn't include the taxes, but we want it to.

Reply:
I updated this to use `get_cart_subtotal` as Storefront is using it: https://github.com/woocommerce/storefront/blob/3.9.1/inc/woocommerce/storefront-woocommerce-template-functions.php#L93

Reply:
I don't think we can use the same function as Storefront here, unfortunately. `get_cart_subtotal()` returns some HTML code and I think we should avoid this because then we can't mimic the same markup easily from the React component.

I saw you used `wp_strip_all_tags()`. That would work, but instead I wonder if we could use: `$cart->get_total( 'edit' );`. This is how it's done in the API, so we will make sure the value we display in the React component and in the PHP button is the same.

https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/a09e165253b4f531864eb48705adb84ea73cdfb4/src/StoreApi/Schemas/CartSchema.php#L372-L373

Reply:
> Is there another way to format the price without HTML tags in WC?

Good point. I couldn't find any. I see WC Admin is using a similar solution:

https://github.com/woocommerce/woocommerce-admin/blob/206384abda75a823c1e85c679c86e4a92b26485a/src/Notes/NewSalesRecord.php#L95

I think we can leave it as-is and open an issue in [WC core repo](https://github.com/woocommerce/woocommerce/issues) explaining that we needed to do this hackish solution and WC Admin is doing it too. This way they can evaluate if it would be worth adding something like that in WC.